src/GitHubApps/src/DarcBot/AzureFunction.cs

```diff
@@ -0,0 +1,212 @@
+            return new OkObjectResult("Success");
+        }
+
+        private static async Task IngestTriageItemsIntoKusto(TriageItem[] triageItems)
+        {
+            _log.LogInformation("Entering IngestTriageItemIntoKusto");
+            string kustoIngestConnectionString = System.Environment.GetEnvironmentVariable("KustoIngestConnectionString");
+            string databaseName = System.Environment.GetEnvironmentVariable("KustoDatabaseName");
+            IKustoIngestClient ingestClient = KustoIngestFactory.CreateQueuedIngestClient(kustoIngestConnectionString);
+            await KustoHelpers.WriteDataToKustoInMemoryAsync(
+                ingestClient,
+                databaseName,
+                "TimelineIssuesTriage",
+                _log,
+                triageItems,
+                b => new[]
+                {
+                    new KustoValue("ModifiedDateTime", b.ModifiedDateTime.ToString(), KustoDataTypes.DateTime),
+                    new KustoValue("BuildId", b.BuildId.ToString(), KustoDataTypes.Int),
+                    new KustoValue("RecordId", b.RecordId.ToString(), KustoDataTypes.Guid),
+                    new KustoValue("Index", b.Index.ToString(), KustoDataTypes.Int),
+                    new KustoValue("UpdatedCategory", b?.UpdatedCategory, KustoDataTypes.String),
+                    new KustoValue("Url", b?.Url, KustoDataTypes.String)
+                });
+        }
+
+        private static string GetDarcBotProperty(string propertyName, string body)
+        {
+            MatchCollection propertyMatches = _darcBotPropertyRegex.Matches(body);
+            foreach (Match propertyMatch in propertyMatches)
+            {
+                if (propertyMatch.Success)
+                {
+                    string key = propertyMatch.Groups["key"].Value;
+                    string value = propertyMatch.Groups["value"].Value;
+                    if (propertyName.Equals(key, StringComparison.OrdinalIgnoreCase))
+                    {
+                        return value;
+                    }
+                }
+            }
+            return null;
+        }
+
+        private static TriageItem GetTriageItemProperties(string body)
+        {
+            TriageItem triageItem = new TriageItem();
+
+            if (!string.IsNullOrEmpty(body))
+            {
+                Match propertyMatch = _darcBotPropertyRegex.Match(body);
+                if (propertyMatch.Success)
+                {
+                    triageItem.UpdatedCategory = GetDarcBotProperty("category", body);
+                }
+
+                Match triageIdentifierMatch = _darcBotIssueIdentifierRegex.Match(body);
+                if (triageIdentifierMatch.Success)
+                {
+                    triageItem.BuildId = int.Parse(triageIdentifierMatch.Groups["buildid"].Value);
```

Comment:
Should we use "TryParse" here?  In case someone puts gibberish in the issue, we probably don't want to crash out the webhook.